servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1424,7 +1424,15 @@ public final Completable subscribeOn(Executor executor) {
      * @param executor {@link Executor} to use.
      * @return A new {@link Completable} that will use the passed {@link Executor} to invoke all methods
      * {@link Subscriber}, {@link Cancellable} and {@link #handleSubscribe(CompletableSource.Subscriber)}.
-     */
+     * @deprecated This operator has been deprecated because of upcoming behavior changes in how the operator does
+     * offloading. Previously the subscribe would be offloaded for the entire subscribe chain, not just the portion
```

Comment:
How about this:

```
@deprecated This operator has been deprecated because of upcoming behavior changes in how
offloading via operators is done. Originally offloading for subscribe/subscription was applied at
the "bottom" of chain (logically the last operator in the chain closest to the subscriber), and
offloading for subscriber was applied at the "top" of the operator chain (logically the first operator
in the chain after the async source). The offloading done was the same regardless of where the
operator was placed in the chain. However, this behavior will soon change to instead apply 
offloading exactly where the offloading operators are applied in the chain. This change in behavior
means that it no longer makes sense to fuse the offloading of publish and subscribe as the location
of the operators in the execution chain is now significant. Publish and subscribe offloading, when
required, will typically be placed in different locations. Use separate subscribeOn(Executor) and
publishOn(Executor) operators instead.
```

Reply:
few more suggestions, lgtm:
- `The offloading done was the same regardless of where the operator was placed in the chain.` -> Therefore the offloading doesn't respect the order where the operator is applied.
- use `{@link ..}` where possible (e.g. `{@link #subscribeOn(Executor)}`, {@link Subscriber}, ..)